src/devices/thermostat.h

```diff
@@ -170,6 +170,7 @@ class Thermostat : public EMSdevice {
     uint8_t ibaClockOffset_       = EMS_VALUE_UINT_NOTSET; // offset (in sec) to clock, 0xff = -1 s, 0x02 = 2 s
 
     uint16_t errorNumber_        = EMS_VALUE_USHORT_NOTSET;
+    char     lastCode_[30]       = {'\0'};
```

Comment:
isn't last code only for the Boiler, not the thermostat?

Reply:
Boiler stores in message 0x10 and 0x11, for my boiler the last is in 0x10, 0x11 is older, but #614 reports last in 0x11. 
Thermostat has it's own storage in message 0x12, mainly communication errors.
In my system (output from this code):
Boiler: `  Last error: 6L(548) 24.01.2020 02:52`
Thermostat: ` Last error: 22(816) 17.11.2020 10:44`